README.md

```diff
@@ -200,11 +200,9 @@ Copyright:
  © 2013-2018 Alexey Khokholov (Nuke.YKT).  
 License: [GPL-2.0+](https://www.gnu.org/licenses/old-licenses/gpl-2.0.html)
 
-Files: `soundfonts/TimGM6mb.sf2`  
+Files: `soundfonts/GeneralUser_GS_v1.51.sf3`  
```

Comment:
Did you mean `GeneralUser_GS_v1.511.sf3` ?

Reply:
Yes, thanks. Fixed in https://github.com/fabiangreffrath/woof/pull/743/commits/edb57a544171867464eafeee22226b327059a456